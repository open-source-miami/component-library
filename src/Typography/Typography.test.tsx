import * as React from "react"
import { Typography } from "index"

it("Typography snapshot", () => {
  expect(<Typography variant="h1">Alex</Typography>).toMatchSnapshot()
})
