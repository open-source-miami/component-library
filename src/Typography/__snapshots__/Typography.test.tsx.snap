// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Typography snapshot 1`] = `
<Typography
  variant="h1"
>
  Alex
</Typography>
`;
